plugins/kubescape.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kubescape
+spec:
+  homepage: https://github.com/kubescape/kubescape/
```

Comment:
Recommend importing `import _ "k8s.io/client-go/plugin/pkg/client/auth"` as the users typically expect plugins to work on cloud clusters.

Reply:
hi @ahmetb can you point out an example?